common/lib/xmodule/xmodule/js/js_test.yml

```diff
@@ -51,7 +51,6 @@ lib_paths:
     - common_static/js/vendor/CodeMirror/codemirror.js
     - common_static/js/vendor/tinymce/js/tinymce/jquery.tinymce.min.js
     - common_static/js/vendor/tinymce/js/tinymce/tinymce.full.min.js
-    - common_static/js/vendor/mathjax-MathJax-c9db6ac/MathJax.js
```

Comment:
Note to reviewers: one of the only 2 changes that aren't strictly a file delete.